packages/types/src/srml/grandpa/definitions.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+export default {
+  types: {
+    AuthorityWeight: 'u64',
+    NextAuthority: '(SessionKey, u64)',
+    PendingPause: {
+      /// Block at which the intention to pause was scheduled.
+      scheduledAt: 'BlockNumber',
+      /// Number of blocks after which the change will be enacted.
+      delay: 'BlockNumber'
+    },
+    PendingResume: {
+      /// Block at which the intention to resume was scheduled.
+      scheduledAt: 'BlockNumber',
+      /// Number of blocks after which the change will be enacted.
+      delay: 'BlockNumber'
+    },
+    StoredPendingChange: {
```

Comment:
Not technically needed, we had a class for this - but since we created grandpa, removed the class and moved the definition in here